packages/contracts/src/Game.sol

```diff
@@ -518,12 +519,14 @@ contract Game {
         if (address(drawVerifier) == address(0)) return;
 
         // construct circuit public signals
-        uint256[68] memory pubSignals;
-        // TODO copy deck in slots 0-63
-        pubSignals[64] = uint256(initialDeckRoot);
-        pubSignals[65] = uint256(pdata.handRoot); // TODO should be supplied as argument
-        pubSignals[66] = committedSalt;
-        pubSignals[67] = randomness;
+        uint256[7] memory pubSignals;
+        pubSignals[0] = 0; // TODO: first field element
+        pubSignals[1] = 0; // TODO: second field element
```

Comment:
The only thing I'm not entirely sure is that should we parse the deck cards into field elements on the smart contract? Or should we refactor the representation of cards in the smart contract as field elements from the beginning?